### ISSUE TITLE
Decoding failure when deleting documents

### ISSUE THREAD

**Issue by Sembiance** [meilisearch/MeiliSearch#1601]
**Describe the bug**
Attempting to delete any document from a dataset below results in an error:
```
  {
    "status": "failed",
    "updateId": 2,
    "type": {
      "name": "DocumentsDeletion",
      "number": 1
    },
    "message": "internal: decoding failed",
    "errorCode": "internal",
    "errorType": "internal_error",
    "errorLink": "https://docs.meilisearch.com/errors#internal",
    "duration": 0.0,
    "enqueuedAt": "2021-08-20T01:05:08.859222412Z",
    "processedAt": "2021-08-20T01:05:08.860058643Z"
  }
```

**To Reproduce**
1. Run `meilisearch` `v0.21.0rc6`
2. Create the index :
`curl -X POST 'http://localhost:7700/indexes' --data '{"uid": "test", "primaryKey": "docid"}'`
`curl -X POST 'http://localhost:7700/indexes/test/settings' --data '{"filterableAttributes": ["label"]}'`
3. Populate the data:
```
curl -X POST 'http://localhost:7700/indexes/test/documents' --data '[
	{"docid":"1_4","label":"sign"},
	{"docid":"1_5","label":"letter"},
	{"docid":"1_7","label":"abstract,cartoon,design,pattern"},
	{"docid":"1_36","label":"drawing,painting,pattern"},
	{"docid":"1_37","label":"art,drawing,outdoor"},
	{"docid":"1_38","label":"aquarium,art,drawing"},
	{"docid":"1_39","label":"abstract"},
	{"docid":"1_40","label":"cartoon"},
	{"docid":"1_41","label":"art,drawing"},
	{"docid":"1_42","label":"art,pattern"},
	{"docid":"1_43","label":"abstract,art,drawing,pattern"},
	{"docid":"1_44","label":"drawing"},
	{"docid":"1_45","label":"art"},
	{"docid":"1_46","label":"abstract,colorfulness,pattern"},
	{"docid":"1_47","label":"abstract,pattern"},
	{"docid":"1_52","label":"abstract,cartoon"},
	{"docid":"1_57","label":"abstract,drawing,pattern"},
	{"docid":"1_58","label":"abstract,art,cartoon"},
	{"docid":"1_68","label":"design"},
	{"docid":"1_69","label":"geometry"}]'
```
4. Attempt to delete `any` document: `curl -X DELETE 'http://localhost:7700/indexes/test/documents/1_4'`
5. Check the status, see it failed: `curl -X GET 'http://localhost:7700/indexes/test/updates'`

**Expected behavior**
I expect it not to fail.

**MeiliSearch version:** v0.21.0rc6
Works fine with v0.20.0

**Additional context**
If you skip modifying the `filterableAttributes` index settings, then it works ok. So it must have something to do with that.
Note, if you change it to `"filterableAttributes": ["doc_id"]` it still fails with the same `internal: decoding failed` error

**Comment by curquiza** [comment]
Hello @Sembiance! Thank you so much for this report and for the detailed steps! 🙏 I think this is the same bug as https://github.com/meilisearch/MeiliSearch/issues/1594 we did not succeed to reproduce!

I followed your steps and I got the same error as you report!

**Comment by curquiza** [comment]
Closed by #1605
